tests/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '3'
+services:
+  ansible:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      CONJUR_APPLIANCE_URL: http://conjur:3000
+      CONJUR_ACCOUNT: cucumber
+      CONJUR_AUTHN_LOGIN: host/ansible/ansible-master
+      CONJUR_AUTHN_API_KEY: ${ANSIBLE_MASTER_AUTHN_API_KEY}
+      COMPOSE_PROJECT_NAME: ${COMPOSE_PROJECT_NAME}
+      CONJUR_VERSION: 5
+      CONJUR_CERT_FILE: ${ANSIBLE_CONJUR_CERT_FILE}
+    volumes:
+      - ../plugins:/root/.ansible/plugins
+      - ..:/cyberark
+      - /var/run/docker.sock:/var/run/docker.sock
+
+  pg:
+    image: postgres:9.4
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
can we use `password` instead and configure conjur with the password? it shouldn't be much harder, and it's slightly more secure (which sets a better example)

Reply:
which i think is really ok when running in test envs. I remember having a conversation about that.

Reply:
we do use it in a number of places, but it is incredibly simple to use a password instead and the security benefit makes it worth it

all you have to do is set these env vars in the pg container
```
POSTGRES_HOST_AUTH_METHOD=password
POSTGRES_PASSWORD=YOUR_PASSWORD_HERE
```

then your connection string in conjur becomes:
```
postgres://postgres:YOUR_PASSWORD_HERE@pg/postgres
```

Reply:
fwiw, I view continuing to propagate `trust` as the auth method as adding debt at this point.